src/electron/electron/ipc/IpcHandler.ts

```diff
@@ -56,14 +71,19 @@ export class IpcHandler {
       obfuscateWindowActivityDtosById: this.obfuscateWindowActivityDtosById,
       startDataExport: this.startDataExport,
       revealItemInFolder: this.revealItemInFolder,
+      openUploadUrl: this.openUploadUrl,
       startAllTrackers: this.startAllTrackers,
       triggerPermissionCheckAccessibility: this.triggerPermissionCheckAccessibility,
       triggerPermissionCheckScreenRecording: this.triggerPermissionCheckScreenRecording
     };
-    LOG.debug('IpcHandler constructor called');
-  }
 
-  public init(): void {
+    if (studyConfig.trackers.taskTracker.enabled) {
```

Comment:
for versions of PA where the taskTracker is unavailable, and thus, the config doesn't contain a related entry, will this just fail "silently"?

Also, I wonder if we should merge this to the official PA-repo, or keep in the fork?

Reply:
ha, good point. I assumed that every config will have this (set to `enabled:false` without the self-reflection compoment). I will make the taskTracker setting optional. And eventually we'll remove it from here completely when we do the fork.

Reply:
As a side note, instead of having the enabled flag on all the other trackers, we could also consider their existence in the config optional. This would simplify the code and the config setup for researchers. It'd be more like a plug-and-play thingy...